src/html5/tokenizer/character_reference.rs

```diff
@@ -6,6 +6,7 @@ use crate::bytes::{
 };
 use crate::html5::error_logger::ParserError;
 use crate::html5::tokenizer::replacement_tables::{TOKEN_NAMED_CHARS, TOKEN_REPLACEMENTS};
+
 use crate::html5::tokenizer::{Tokenizer, CHAR_REPLACEMENT};
```

Comment:
What about keeping the `use` statements together, just so that the linter can do its thing when it organizes them?

Reply:
I'm for it, but I think this is a IDE thing that doesn't know how to set them correctly..  I will try and look out for this